pkg/messaging/bots.go

```diff
@@ -1,6 +1,10 @@
 package messaging
 
+import "go.nanomsg.org/mangos/v3/protocol"
+
 type Bot interface {
 	SendMessage(msg []byte)
 	Start()
+	SubscribeToAllAlerts() error
+	SetPubSubSocket(pubSubSocket protocol.Socket)
 }
```

Comment:
What's the purpose of this interface?

Reply:
`StartPubSubMessagingClient` and other messaging services are supposed to be common for all possible bots. They are supposed to receive an array of interfaces `[]Bot` and call corresponding functions like `SendMessage`. They shouldn't know details of every bot. That is why some functions are moved to the interface